scripts/rebuild_all.sh

```
@@ -33,4 +33,6 @@ oc annotate components/validation-$version build.appstudio.openshift.io/request=
 oc annotate components/virt-v2v-rhel9-$version build.appstudio.openshift.io/request=trigger-pac-build
 oc annotate components/vsphere-copy-offload-populator-$version build.appstudio.openshift.io/request=trigger-pac-build
 oc annotate components/vsphere-xcopy-volume-populator-$version build.appstudio.openshift.io/request=trigger-pac-build
+oc annotate components/deep-inspection-$version build.appstudio.openshift.io/request=trigger-pac-build
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)